users/urls.py

```diff
@@ -20,6 +20,8 @@
     path('refresh_token/', TokenRefreshView.as_view(), name='refresh_token'),
 
     path('profile_update/', views.UserProfileUpdate.as_view(), name="profile_update"), 
+    path('profile_update/skills/', views.update_skills, name='update_skills'),
+    path('profile_update/interests/', views.update_interests, name='update_interests'),
```

Comment:
Shouldn't be this be contemplated in the `UserProfileUpdate` class?

Reply:
I wasn't sure how to get this all setup using nested relationships.  It seems like the serializer kept complaining because skills is a nested array.

Reply:
Oh ok. Yeah, it can be tricky to use nested serializers but there is probably a way to make it work. But if the maintainers are ok with these new functions it can be done this way.